### PR TITLE
chore(agw): Update metrics naming on exporting 

### PR DESCRIPTION
**Pull request by ardzoht**
## Summary

- Currently, metrics collector on magmad is using a deprecated proto file (`metricsd.proto`) to set the name for each Metric family (collection of metrics on a service)
- This change updates the filling of the metric name to use the name from the prometheus metric name rather from the protobuf description name. 
- Updates s1ap tests to remove `metricsd` proto enum values

## Test Plan

- `make integ_test`
- running local environment and making sure metrics are still collected and exported successfully to the cloud:
```
Feb 08 22:45:33 magma-dev-focal magmad[51255]: DEBUG:root:Collected 9 from state...
Feb 08 22:45:33 magma-dev-focal magmad[51255]: DEBUG:root:Collected 18 from pipelined...
Feb 08 22:45:33 magma-dev-focal magmad[51255]: DEBUG:root:Metrics upload success for service pipelined (chunk 0)
Feb 08 22:45:33 magma-dev-focal magmad[51255]: DEBUG:root:Metrics upload success for service magmad (chunk 0)
Feb 08 22:45:33 magma-dev-focal magmad[51255]: DEBUG:root:Metrics upload success for service sessiond (chunk 0)
Feb 08 22:45:33 magma-dev-focal magmad[51255]: DEBUG:root:Metrics upload success for service state (chunk 0)
Feb 08 22:45:33 magma-dev-focal magmad[51255]: DEBUG:root:Metrics upload success for service enodebd (chunk 0)
```

- `service303_cli.py metrics pipelined` before:
```
name: "50"
type: GAUGE
metric {
  label {
    name: "10"
    value: "3.8.5"
  }
  label {
    name: "11"
    value: "CPython"
  }
  label {
    name: "9"
    value: "3"
  }
  label {
    name: "8"
    value: "8"
  }
  label {
    name: "7"
    value: "5"
  }
  gauge {
    value: 1.0
  }
  timestamp_ms: 1644366222080
}

name: "309"
type: COUNTER

name: "550"
type: COUNTER
metric {
  counter {
    value: 2.0
  }
  timestamp_ms: 1644366222080
}

name: "350"
type: COUNTER

name: "351"
type: COUNTER
metric {
  counter {
    value: 0.0
  }
  timestamp_ms: 1644366222080
}

name: "352"
type: COUNTER

name: "353"
type: COUNTER
metric {
  counter {
    value: 0.0
  }
  timestamp_ms: 1644366222080
}
```
- `service303_cli.py metrics pipelined` after:
```
name: "python_info"
type: GAUGE
metric {
  label {
    name: "10"
    value: "3.8.5"
  }
  label {
    name: "11"
    value: "CPython"
  }
  label {
    name: "9"
    value: "3"
  }
  label {
    name: "8"
    value: "8"
  }
  label {
    name: "7"
    value: "5"
  }
  gauge {
    value: 1.0
  }
  timestamp_ms: 1644374679693
}

name: "streamer_responses"
type: COUNTER

name: "service_errors"
type: COUNTER
metric {
  counter {
    value: 2.0
  }
  timestamp_ms: 1644374679693
}

name: "dp_send_msg_error"
type: COUNTER

name: "arp_default_gw_mac_error"
type: COUNTER
metric {
  counter {
    value: 0.0
  }
  timestamp_ms: 1644374679693
}

name: "openflow_error_msg"
type: COUNTER

name: "unknown_pkt_direction"
type: COUNTER
metric {
  counter {
    value: 0.0
  }
  timestamp_ms: 1644374679693
}

name: "enforcement_rule_install_fail"
type: COUNTER

name: "enforcement_stats_rule_install_fail"
type: COUNTER
```

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]

File: orc8r/gateway/python/magma/magmad/tests/BUILD.bazel [hunk 1]
```diff
@@ -29,12 +29,6 @@ pytest_test(
     deps = ["//orc8r/gateway/python/magma/magmad:magmad_lib"],
 )
 
-pytest_test(
-    name = "metrics_tests",
-    srcs = ["metrics_tests.py"],
```
Comment (themarwhal) [inline orc8r/gateway/python/magma/magmad/tests/BUILD.bazel:52]: is this a deleted test?

Comment (themarwhal) [reply orc8r/gateway/python/magma/magmad/tests/BUILD.bazel:52]: oh i see it below :p

File: lte/gateway/python/integ_tests/s1aptests/test_guti_attach.py [hunk 2]
```diff
@@ -0,0 +1,220 @@
+"""
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_guti_attach.py]: what is the reason for file name change?

Comment (ulaskozat) [reply lte/gateway/python/integ_tests/s1aptests/test_guti_attach.py]: what is the difference from `test_guti_attach_with_zero_mtmsi.py`

Comment (ulaskozat) [reply lte/gateway/python/integ_tests/s1aptests/test_guti_attach.py]: I do not see the connection of this test to the metrics naming.

Comment (ardzoht) [reply lte/gateway/python/integ_tests/s1aptests/test_guti_attach.py]: i think this file got included as part of a rebase with other commit I had on my local history, I will remove it, this is not supposed to be added

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**